code/datums/looping_sounds/item_sounds.dm

```diff
@@ -34,4 +34,14 @@
 
 #undef RAD_GEIGER_LOW
 #undef RAD_GEIGER_MEDIUM
-#undef RAD_GEIGER_HIGH
\ No newline at end of file
+#undef RAD_GEIGER_HIGH
```

Comment:
move these #undef at the bottom of file

Reply:
they are below the geiger counter code, they cease to be relevant once it ends. I don't remember where but I did it this way because another file did it that way